update.py

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i python3 -p git python3 python3Packages.pyyaml python3Packages.parse nix-prefetch-git
+#! nix-shell -i python3 -p git python3 python3Packages.pyyaml python3Packages.parse nix-prefetch-git nix-prefetch-github
```

Comment:
this dependency was missing. It's used for instance here https://github.com/poscat0x04/hls-nix/pull/13/files#diff-3db03b056e588432b22814fac93c11d44c8a4d8d958713cd2bf70b1d1d0ab48dR39